docs/DEVELOPER.md

```diff
@@ -24,28 +28,44 @@ The scripts referenced below can be used as long as the root .env file is set
 5. `ts-node src/scripts/download-new-songs` can be used to download the remaining songs in the database
 
-## Debug Mode
-Having every song downloaded may be infeasible for local development. A debug mode can be activated by modifying `src/config/debug_settings.json`, and running in development mode. `forcedSongId` will force the bot to play a specific song given its ID. If `null`, will ignore this setting. `skipSongPlay` will start a game session with a song even if it is not downloaded on the local machine. 
+## Debug Mode - deprecated
+
+Having every song downloaded may be infeasible for local development. A debug mode can be activated by modifying `src/config/debug_settings.json`, and running in development mode. `forcedSongId` will force the bot to play a specific song given its ID. If `null`, will ignore this setting. `skipSongPlay` will start a game session with a song even if it is not downloaded on the local machine.
 
 ## Creating Migrations
+
 In order to modify the database schema, you need to create a migration file. This can be done using `npx knex migrate:make migration_name --knexfile src/config/knexfile_kmq.js`. Modify the created migration file to describe the change in schema. [Details here](http://knexjs.org/#Migrations).
 
 A migration can be undone using `knex migrate:down migration_name.js --knexfile src/config/knexfile_kmq.js`
 
 ## Process Management
-KMQ is hosted using PM2 as a process manager. Some scripts may assume that KMQ is being managed by `pm2` under the name `kmq`. 
+
+KMQ is hosted using PM2 as a process manager. Some scripts may assume that KMQ is being managed by `pm2` under the name `kmq`.
 
 ## Scripts
-Located under `src/scripts`  
-- `announce-restart.ts [n]`: Set up a timer for `n` minutes before the bot is stopped. Each server currently playing a game will be notified at regular intervals until the shutdown  
-- `download-new-songs.ts {n}`: If a parameter isn't specified, will download every song not currently downloaded. If specified, will download the top `n` most viewed songs according to the database.  
-- `get-unclean-song-names.ts`: Returns a list of potentially unusual song names (hangul, hiragana, strange punctuation) to inspect. Used to check for songs that need aliasing.  
+
+Located under `src/scripts`
+
+- `announce-restart.ts [n]`: Set up a timer for `n` minutes before the bot is stopped. Each server currently playing a game will be notified at regular intervals until the shutdown
+- `download-new-songs.ts {n}`: If a parameter isn't specified, will download every song not currently downloaded. If specified, will download the top `n` most viewed songs according to the database.
+- `get-unclean-song-names.ts`: Returns a list of potentially unusual song names (hangul, hiragana, strange punctuation) to inspect. Used to check for songs that need aliasing.
+
+## Mac specific installation help
```

Comment:
No reason to have Mac specific instructions, users are expected to know how to install these basic packages  from the prerequisites section